src/commands/build.rs

```diff
@@ -2,17 +2,29 @@ use std::path::Path;
 
 use clap::ArgMatches;
 
-use crate::build;
-use crate::settings::toml::Manifest;
-
 use super::DEFAULT_CONFIG_PATH;
+use crate::build::build_target;
+use crate::settings::toml::Manifest;
+use crate::terminal::message::{Message, StdOut};
 
-pub fn run(matches: &ArgMatches) -> Result<(), failure::Error> {
+pub fn build(matches: &ArgMatches) -> Result<(), failure::Error> {
     log::info!("Getting project settings");
     let config_file = matches.value_of("config").unwrap_or(DEFAULT_CONFIG_PATH);
     let config_path = Path::new(config_file);
     let manifest = Manifest::new(&config_path)?;
     let env = matches.value_of("env");
     let target = &manifest.get_target(env, false)?;
-    build(&target)
+    let build_result = build_target(&target);
+    match build_result {
+        Ok(Some(msg)) => {
+            StdOut::success(&msg);
+            Ok(())
+        }
+        Ok(None) => {
```

Comment:
actually this match seems unnecessary. i think i would prefer that build target just return `Result<String>` rather than `Result<Option<String>>` and backfilling the none  as  it does here.